buildbot/buildbot/worker.py

```diff
@@ -315,10 +330,11 @@ def stage_hls(db, config):
         hw_basename, hw_c, hw_o = _hw_filenames(task)
         _task_config(task, config)
         flags = shlex.split(task['sdsflags'])
+        sds_cmd = _sds_cmd(prefix, hw_basename, hw_c, task) 
```

Comment:
If `task` is going to be an argument to `_sds_cmd`, then the `hw_basename` and `hw_c` parameters can also just come straight from `task` (and get looked up within there). And we should probably also just append the flags in there, right?

Reply:
Yes, thanks!